awsmobilejs/backend/cloud-api/notify/app.js

```diff
@@ -21,9 +21,9 @@ const hasDynamicPrefix = true;
 const userIdPresent = false;
 const partitionKeyName = "id";
 const partitionKeyType = "S"
-const sortKeyName = "time";
-const sortKeyType = "N";
-const hasSortKey = true;
+const sortKeyName = "";
```

Comment:
I feel like sortKeyName should be "scheduled_at".  We can change that down the line though.